packages/pwa/.eslintrc.js

```diff
@@ -0,0 +1,8 @@
+/*
+ * Copyright (c) 2021, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+require.resolve('pwa-kit-build/configs/eslint/eslint-config')
```

Comment:
[`require.resolve`](https://nodejs.org/docs/latest-v14.x/api/modules.html#modules_require_resolve_request_options) just returns the resolved path to the file. You'll still need to import it to use it.

Reply:
ah I see, I made the change. Please have another look, thanks!